pyproject.toml

```diff
@@ -180,25 +180,25 @@ reports = false
 score = false
 
 [tool.pylint.format]
-indent-string = '    '
 indent-after-paren = 4
+indent-string = '    '
```

Comment:
I wonder why this wasn't caught before...

Reply:
What exactly? This is the same as it was on line 183, but the order was moved in toml.

Reply:
Yeah, I wonder why pyproject-fmt didn't catch any of this before.